internal/restic/node_xattr.go

```diff
@@ -16,6 +17,10 @@ import (
 // Getxattr retrieves extended attribute data associated with path.
 func Getxattr(path, name string) ([]byte, error) {
 	b, e := xattr.Get(path, name)
+	if err, ok := e.(*xattr.Error); ok && err.Err == syscall.ENODATA {
+		debug.Log("Getxattr error:ENODATA, path:%s", path)
```

Comment:
I don't think the `debug.Log` is necessary.

Reply:
done